src/com/company/FibonacciExecute.java

```diff
@@ -0,0 +1,30 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class FibonacciExecute {
+
+    public static int execute(int numeroFibo) {
```

Comment:
numeroFibo? I think that this variable represent the fibonacci number in position or if you search about it, you could rename this variable to "fibonacciCardinal". In spanish you could search "cardinales y ordinales" to understand what I said